.github/workflows/check.yml

```diff
@@ -138,7 +138,7 @@ jobs:
 
     - name: Configure flatpak-builder
       # TODO: Enable gtk-doc builds
-      run: meson . ${BUILDDIR}
+      run: meson setup . ${BUILDDIR}
```

Comment:
I didn't even know it can take two args, but my meson tells me it's `meson setup ... [builddir] [sourcedir]`.

Reply:
Yea looks like it was switched at some point.